### PR TITLE
fix(transformers): fix bugs for transformers upgrade

### PR DESCRIPTION
**Pull request by wcrzlh**
# What does this PR do?

Fixes # (issue)
- [x] fix dtype unsupport probelm of "Tensor.norm" in bf16 in CLIP Model
- [x] fix random initial weights precision bug
- [x] fix mint.softmax api interface bug(axis-->dim)
- [x] add more model import of gpt2 for mindone.diffusers usage     
- [x] remove debug utils
- [x] upgrade API interface: _has_non_default_generation_parameters 

Adds # (feature)
- [x] set transformers version to 4.46.3 for downloading

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [ ] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [ ] Did you build and run the code without any errors?
- [ ] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@xxx

### CODE REVIEW

**Review by CaitinZhao — APPROVED** [review]

File: mindone/transformers/models/clip/modeling_clip.py
Comment (SamitHuang) [inline mindone/transformers/models/clip/modeling_clip.py:1]: still compatible with ms2.3.1?

Comment (wcrzlh) [reply mindone/transformers/models/clip/modeling_clip.py:1]: For CLIP model, it would not be compatible with ms 2.3.1. It is compatible with ms 2.5.0.
Reason1: Mint.nn.Conv2d change: https://github.com/mindspore-lab/mindone/pull/883/files#diff-001a2f13ca880bb64e1513e6c044426f80bf45e7d577e84706fafbf7b3caa0d2R166
Reason2: mint.norm in this pr: this is for solving the problem that "Tensor.norm" do not support bf16. This problem still happens in ms2.3.1

**Review by vigo999 — APPROVED** [review]
